sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubExtensionConfigProvider.cs

```diff
@@ -95,7 +96,7 @@ internal static void LogExceptionReceivedEvent(ExceptionReceivedEventArgs e, ILo
             Utility.LogException(e.Exception, message, logger);
         }
 
-        private IAsyncCollector<EventData> BuildFromAttribute(EventHubAttribute attribute)
+        private EventHubAsyncCollector BuildFromAttribute(EventHubAttribute attribute)
```

Comment:
Note to self:  Revert this if we decide to move forward with the extension method